### ISSUE TITLE
Slide to unlock exists even with password set

### ISSUE THREAD

**Issue by alphamu** [cm7sgt/android_device_samsung_galaxytab#16]
I have used Set Location & Security to set a password on my phone.
When I lock the screen and wake it again, I see the slide to unlock screen.
I believe this screen should not show if a password is set.

The slide to unlock screen does not take up the full screen.

The password screen seems to time out even if I'm interacting with it, though this one I have not been able to reliably reproduce.
I'll upload screenshots for the slide problem and showing that both screens are in fact showing up. http://customizedby.me/screen_lock_problems.png

In the first screen with the slider, for some reason instead of showing the BG it's showing the icons, however this is only a problem with the screen capture. In reality I see the background.

**Comment by cm7sgt** [comment]
The rotary lockscreen dimensions have been fixed by Fabrizio and will be included in next release. As for how it interacts with the lockscreen it is most likely an upstream matter not related to porting CM7 to Galaxy Tab.

**Comment by alphamu** [comment]
I know you said this looks like an upstream issue. But I just noted that when I set a password lock, the slide to unlock appears. When I set a pin lock, the slide to lock appears. However when I set a pattern lock, the slide to unlock does not appear.

Also, in the settings when setting up a pattern lock, the images of the circle and dots seem misaligned. The dots are to the top left of the circle instead of being dead centre inside them.
